docs/source/getting-started/docker-machine-drivers.adoc

```diff
@@ -108,6 +108,26 @@ For more information, see the GitHub documentation of the https://github.com/zch
 
 To manually install the xhyve driver, run the following commands:
 
+. Download the xhyve binary using:
```

Comment:
This is not `xhyve` binary but it's `docker-machine-driver-xhyve` binary both are different.